tests/test_xml_files/partial.xml

```diff
@@ -35,8 +35,6 @@
         <voice>1</voice>
         <type>eighth</type>
       </note>
-    </measure>
-    <measure number="1">
```

Comment:
:x: This is definitely incorrect. The original content was correct (or at least renders in the desired way in MuseScore).